src/script/conversation/ConversationRepository.ts

```diff
@@ -1351,6 +1357,11 @@ export class ConversationRepository {
     }
   };
 
+  private readonly markConversationReadOnly = async (conversationEntity: Conversation) => {
```

Comment:
Let's make this method take `CONVERSATION_READONLY_STATE` as an argument also.

Reply:
fixed [commit](https://github.com/wireapp/wire-webapp/pull/15759/commits/4ece444e65cd7aa0718c3c2fc4630a3e4a64c66f)